packages/components/pie-modal/test/visual/pie-modal.spec.ts

```diff
@@ -414,3 +414,33 @@ test.describe('`position`', () => {
         });
     });
 });
+
+test.describe('Prop: `hasStackedActions`', () => {
+    test.describe('when true', () => {
+        (['small', 'medium', 'large'] as Array<ModalProps['size']>)
+            .forEach((size) => {
+                test(`actions should be full width (at narrow viewports) for a modal with size = ${size}`, async ({ page, mount }) => {
```

Comment:
Would you mind changing this to `should ....` maybe `should display actions at full width ....`

Reply:
Fixed